_developer-files/connect/api/objects/form-properties/sources/saas/pardot-object.md

```diff
@@ -0,0 +1,50 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/connect-templates/destination-form-property/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#        CONTENT TYPE        #
+# -------------------------- #
+
+product-type: "connect"
+content-type: "api-form"
+form-type: "source"
+key: "source-form-properties-pardot-object"
+
+
+# -------------------------- #
+#        OBJECT INFO         #
+# -------------------------- #
+
+title: "Pardot Source Form Property"
+api-type: "platform.pardot"
+display-name: "Pardot"
+
+source-type: "saas"
+docs-name: "pardot" # This should be whatever integration.name is. Ex: LinkedIn Ads is linkedin-ads
+
+
+# -------------------------- #
+#      OBJECT ATTRIBUTES     #
+# -------------------------- #
+
+uses-start-date: true
+
+# Only source-specific attributes need to be listed here.
+# The following attributes are considered common,
+# and therefore don't need to be listed:
+# anchor_time, cron_expression, frequency_in_minutes, image_version, start_date 
+
+object-attributes:
+  - name: "user_key"
+    type: "string"
+    required: true
+    description: "32-character hexadecimal user key for your user account."
```

Comment:
This description should be something meaningful to the user, and if necessary, provide info and resources about what it is + how to obtain it.

You describe what this is in the setup guide - start there for the description, and include a link to the instructions for retrieving it.